bootstrap/bootstrapper.yaml

```diff
@@ -0,0 +1,70 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  name: kubeflow-cluster-admin
+subjects:
+  - kind: ServiceAccount
+    # Reference to upper's `metadata.name`
+    name: default
+    # Reference to upper's `metadata.namespace`
+    namespace: default
```

Comment:
I had to change the namespace to 'kube-system' otherwise it errors. See below


$ kubectl create ns kubeflow
$ kubectl create -f bootstrapper.yaml
and got the following errors

$ kubectl logs kubeflow-bootstrapper-0
{"filename":"app/server.go:194","level":"info","msg":"Creating namespace: kubeflow for all kubeflow resources","time":"2018-05-09T13:11:50Z"}
{"filename":"bootstrap/main.go:46","level":"fatal","msg":"namespaces is forbidden: User \"system:serviceaccount:kube-system:default\" cannot create namespaces at the cluster scope: Unknown user \"system:serviceaccount:kube-system:default\"\n","time":"2018-05-09T13:11:50Z"}

Tried on GKE
$ ☞  kubectl version
Client Version: version.Info{Major:"1", Minor:"10", GitVersion:"v1.10.0", GitCommit:"fc32d2f3698e36b93322a3465f63a14e9f0eaead", GitTreeState:"clean", BuildDate:"2018-03-27T00:13:02Z", GoVersion:"go1.9.4", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"9+", GitVersion:"v1.9.6-gke.1", GitCommit:"cb151369f60073317da686a6ce7de36abe2bda8d", GitTreeState:"clean", BuildDate:"2018-04-07T22:06:59Z", GoVersion:"go1.9.3b4", Compiler:"gc", Platform:"linux/amd64"}